toxcore/bin_pack.h

```diff
@@ -46,7 +63,7 @@ typedef bool bin_pack_array_cb(Bin_Pack *bp, const Logger *logger, const void *a
  * @retval UINT32_MAX in case of errors such as buffer overflow.
  */
 non_null(1) nullable(2, 3)
-uint32_t bin_pack_obj_size(bin_pack_cb *callback, const Logger *logger, const void *obj);
+uint32_t bin_pack_obj_size(bin_pack_cb *callback, const void *obj, const Logger *logger);
```

Comment:
Re-arrange `@param` in function doc too?

Reply:
Done.

Reply:
I don't see it being fixed, looks like nothing has changed. Did you forget to stage the hunk or something?

Reply:
Fixed now :).